bin/unleash.js

```diff
@@ -25,9 +31,9 @@ const argv = require('yargs')
 
 serverImpl
     .start(argv)
-    .then(conf =>
+    .then(() =>
         console.log(
-            `Unleash started on http://localhost:${conf.app.get('port')}`
+            `Unleash started on http://${argv.host || '127.0.0.1'}:${argv.port}`
```

Comment:
can we get `host` and `port` from the actual server instance instead of from `argv`??

Reply:
yes if we rewrite `createApp` in [server-impl.js](https://github.com/Unleash/unleash/blob/master/lib/server-impl.js#L15) to be async and not resolve before server.listen callback triggers.

Reply:
seems doable as `start` already returns a promise